app/src/main/java/com/example/indoor_volleyball/Activities/CreateGymActivity.java

```diff
@@ -115,6 +127,16 @@ public void onClick(View v) {
         }
     }
 
+    private void setTextOrHide(Object text, TextView view, int formatId) {
+        if (text != null) {
+            view.setVisibility(View.VISIBLE);
+            String formattedString = getString(formatId, text.toString());
+            view.setText(formattedString);
+        } else {
+            view.setVisibility(View.GONE);
+        }
+    }
```

Comment:
You could make a static class called Utils and put this function there (you'll need to make it also take a context so you can call context.getString though!  Also the method will need to be static. ).  This way you can use it in other classes too.